src/main/java/econo/buddybridge/member/controller/MemberController.java

```diff
@@ -0,0 +1,23 @@
+package econo.buddybridge.member.controller;
+
+import econo.buddybridge.member.entity.Member;
+import econo.buddybridge.member.service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+// 수정 필요
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/user")
+public class MemberController {
+    private final MemberService memberService;
+
+    @PostMapping()
+    public String testMember(@RequestBody Member member){
```

Comment:
테스트용 컨트롤러긴 하지만 `@RequestBody`는 API 명세에 의존적이니 DTO를 만들어서 받고 서비스로 넘겨주는건 어떤가요?
```suggestion
    public String testMember(@RequestBody MemberDto member){
```
```java
@Builder
public record MemberDto (
        Long memberId,
        String nickname,
        String email,
        Integer age,
        String profileImageUrl
) {
}
```